src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -24,308 +24,265 @@ public interface TombsOfAmascutConfig extends Config
-		position = 7
+	@ConfigSection(
+		name = "Path of Scabaras",
+		description = "Options for the puzzles in the Path of Scabaras.",
+		position = 3,
+		closedByDefault = true
 	)
-	default boolean showUpdateMessages()
-	{
-		return true;
-	}
+	String SECTION_SCABARAS = "sectionScabaras";
 
-	@ConfigItem(
-		keyName = "hideFadeTransition",
-		name = "Hide Fade Transition",
-		description = "Hides the fade transition between loading zones.",
-		position = 9
+	@ConfigSection(
+		name = "Burial Tomb",
+		description = "Configuration for the burial tomb.",
+		position = 4,
+		closedByDefault = true
 	)
-	default boolean hideFadeTransition()
-	{
-		return false;
-	}
+	String SECTION_BURIAL_TOMB = "sectionBurialTomb";
 
 	@ConfigSection(
 		name = "Invocation Presets",
 		description = "Save presets of invocations to quickly restore your invocations between runs of different types.",
-		position = 100
+		position = 5,
+		closedByDefault = true
 	)
 	String SECTION_INVOCATION_PRESETS = "invocationPresetsSection";
 
-	@ConfigItem(
-		keyName = "invocationPresetsEnable",
-		name = "Enable Presets",
-		description = "Allows for saving and restoring of invocation presets. Right-click \"Preset\" button to save/load.",
-		section = SECTION_INVOCATION_PRESETS,
-		position = 101
+	@ConfigSection(
+		name = "Invocation Screenshot",
+		description = "All config options related to the Invocation Screenshot functionality",
+		position = 6,
+		closedByDefault = true
 	)
-	default boolean invocationPresetsEnable()
-	{
-		return false;
-	}
+	String SECTION_INVOCATION_SCREENSHOT = "invocationScreenshotSection";
 
-	@ConfigItem(
-		keyName = "invocationPresetsScroll",
-		name = "Auto-Scroll",
-		description = "Automatically scroll to invocations which need to be changed for the current preset.",
-		section = SECTION_INVOCATION_PRESETS,
-		position = 102
+	@ConfigSection(
+		name = "Points Tracker",
+		description = "<html>Tracks points for the raid, used in calculating drop chance." +
+			"<br/>NOTE: For teams, you MUST use the RuneLite Party plugin to receive team drop chance.</html>",
+		position = 7,
+		closedByDefault = true
 	)
-	default boolean invocationPresetsScroll()
-	{
-		return true;
-	}
+	String SECTION_POINTS_TRACKER = "sectionPointsTracker";
 
 	@ConfigSection(
-		name = "Invocation Screenshot",
-		description = "All config options related to the Invocation Screenshot functionality",
-		closedByDefault = true,
-		position = 200
+		name = "Time Tracking",
+		description = "Time tracking and splits.",
+		position = 8,
+		closedByDefault = true
 	)
-	String SECTION_INVOCATION_SCREENSHOT = "invocationScreenshotSection";
+	String SECTION_TIME_TRACKING = "sectionTimeTracking";
+
+	@ConfigSection(
+		name = "Miscellaneous",
+		description = "Miscellaneous configurations.",
+		position = 9,
+		closedByDefault = true
+	)
+	String SECTION_MISCELLANEOUS = "sectionMiscellaneous";
```

Comment:
I'd like these to stay at the top and not be folded into a category.

Or perhaps what might be better is to make it the first category and `closedByDefault = false`.